### PR TITLE
imx8mn-ddr4-evk: add machine

### PR DESCRIPTION
**Pull request by vanmaegima**

### CODE REVIEW

**Comment by vanmaegima** [comment]
@ricardosalveti can you do a quick check before I close my board?

**Comment by vanmaegima** [comment]
Tested by fusing and closing my board.

File: meta-lmp-bsp/recipes-support/mfgtool-files/mfgtool-files_%.bbappend [hunk 1]
```diff
@@ -128,3 +134,15 @@ do_deploy:prepend:imx8mp-lpddr4-evk-sec() {
...
+do_compile:append:imx8mn-ddr4-evk-sec() {
+    sed -i 's/imx-boot.*/&.signed/g' bootloader.uuu
+}
+
+do_deploy:prepend:imx8mn-ddr4-evk-sec() {
+    install -d ${DEPLOYDIR}/${PN}
+    install -m 0644 ${WORKDIR}/fuse.uuu ${DEPLOYDIR}/${PN}/fuse.uuu
+    install -m 0644 ${WORKDIR}/close.uuu ${DEPLOYDIR}/${PN}/close.uuu
+    install -m 0644 ${WORKDIR}/readme.md ${DEPLOYDIR}/${PN}/readme.md
+
```
Comment (ricardosalveti) [inline meta-lmp-bsp/recipes-support/mfgtool-files/mfgtool-files_%.bbappend:147]: Please drop this empty line.

**Review by ricardosalveti — APPROVED** [review]
One minor comment, but LGTM

Also please remember to propose changes to our ci (lmp) and website dropdown list once this is merged.

**Pull request merged**